cmd/wave/main.go

```diff
@@ -86,3 +91,15 @@ func envVarName(n string) string {
 	envVar := strings.ToUpper(strings.ReplaceAll(n, "-", "_"))
 	return fmt.Sprintf("%s_%s", envVarNamePrefix, envVar)
 }
+
+func getEnvBool(k string) bool {
+	raw, found := os.LookupEnv(k)
+	if !found {
+		return false
+	}
+	v, err := strconv.ParseBool(strings.ToLower(raw))
+	if err != nil {
+		panic(err)
```

Comment:
I was thinking about this in "fail fast way". Meaning, if you fail to parse the configuration, I'd throw an error ASAP. I consider maybe safer, than using some fallback value. Do you want me to change it? If yes, then what should be the fallback value, in case it fails to parse (I'd assume false).